org.eclipse.m2e.pde.connector/lifecycle-mapping-metadata.xml

```diff
@@ -51,6 +51,53 @@
 				</configurator>
 			</action>
 		</pluginExecution>
+        <pluginExecution>
+            <pluginExecutionFilter>
+                <groupId>org.eclipse.tycho</groupId>
+                <artifactId>tycho-packaging-plugin</artifactId>
+                <versionRange>[1.0.0,)</versionRange>
+                <goals>
+                    <goal>package-plugin</goal>
+                    <goal>package-feature</goal>
+                </goals>
+            </pluginExecutionFilter>
+            <action>
+                <configurator>
+                    <id>org.apache.m2e.pde.connector.configurator.tycho-java</id>
```

Comment:
```suggestion
                    <id>org.apache.m2e.pde.connector.configurator.tycho</id>
```
The plugin.xml does not declare a configurator with that id. Either this Id is wrong or the `plugin.xml`.